test/tick-processor/test-tick-processor-polyfill-brokenfile.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+if (!common.enoughTestCpu)
+  common.skip('test is CPU-intensive');
+
+if (common.isCPPSymbolsNotMapped) {
+  common.skip('C++ symbols are not mapped for this os.');
+}
+
+//This test will produce a broken profile log.
+//ensure prof-polyfill not stuck in infinite loop
+//and success process
```

Comment:
Nit: please use a space at the beginning of each line in a comment.

Reply:
Thank you for the advice.It should have a space at the beginning of each line.